source/content/organizing_and_more/github_org_elements.md

```diff
@@ -1,3 +1,63 @@
-## GitHub's organization structure
+## <i class="fab fa-github"></i> **GitHub**'s Organization Structure
+
+### Key Elements
+
+{% if page %}::::{tabs}{% endif %}
+{% if page %}:::{tab}{% else %}:::{card}{% endif%} Owners
```

Comment:
Hm, is `Owner` not more of a role a User or an Organization can have?
I would also add 'User' as key element, or what do you think?

For GitLab, I ended up with this: 

https://github.com/t4d-gmbh/git-and-its-remotes/blob/4e39fbb69cd0bd152badaa4de064ceb6534bb985/source/content/organizing_and_more/gitlab_key_elements.md?plain=1#L4

https://github.com/t4d-gmbh/git-and-its-remotes/blob/4e39fbb69cd0bd152badaa4de064ceb6534bb985/source/content/organizing_and_more/gitlab_key_elements.md?plain=1#L14

https://github.com/t4d-gmbh/git-and-its-remotes/blob/4e39fbb69cd0bd152badaa4de064ceb6534bb985/source/content/organizing_and_more/gitlab_key_elements.md?plain=1#L27

https://github.com/t4d-gmbh/git-and-its-remotes/blob/4e39fbb69cd0bd152badaa4de064ceb6534bb985/source/content/organizing_and_more/gitlab_key_elements.md?plain=1#L38

Reply:
good point... I'll update it accordingly.